Deploy/configparser.cpp

```diff
@@ -1101,7 +1101,7 @@ bool ConfigParser::initQmake() {
                 return initQmakePrivate(proc);
             }
 
-            QuasarAppUtils::Params::log("You Application requeriment Qt libs but the qmake not findet by option 'qmake' and RPATH."
+            QuasarAppUtils::Params::log("You Application requires Qt libs but the qmake not found by option 'qmake' and RPATH."
```

Comment:
```suggestion
            QuasarAppUtils::Params::log("You Application requires Qt libs but qmake cannot be found in option 'qmake' or PATH."
```

Reply:
@pzhlkj6612   nice work!

Reply:
Thanks. Will add them ASAP

Reply:
OK, resolved.

Reply:
```diff
- ... and RPATH
+ ... and PATH
```

Sorry, this fix seems to wrong. Hi @EndrII , does `ConfigParser::getQtPathesFromTargets()` check the `RPATH` of the executable?

Reply:
The CQtDeployer tool have two message about RPATH and PATH. This change is outdated because all messages outputs will be changed in this [PR](https://github.com/QuasarApp/CQtDeployer/pull/583).

Reply:
Resolved in https://github.com/QuasarApp/CQtDeployer/pull/583/commits/b0e6fe37967ada972b642c1c98271f121f2a0fcf .